src/main/kotlin/com/perquizz/perquizz/migrations/repository/MigrationRepositoryImpl.kt

```diff
@@ -0,0 +1,23 @@
+package com.perquizz.perquizz.migrations.repository
+
+import com.perquizz.perquizz.migrations.dtos.MigrationDto
+import org.flywaydb.core.Flyway
+import org.springframework.stereotype.Repository
+
+@Repository
+class MigrationRepositoryImpl(val flyway: Flyway) : MigrationRepository {
+    override fun findPendingMigrations(): List<MigrationDto> {
+        return flyway.info().pending().map {
+            MigrationDto(
+                it.description,
+                it.script,
+                it.physicalLocation,
+                it.version.name,
+            )
+        }
+    }
+
+    override fun migrate() {
```

Comment:
Firula do kotlin kkk
```suggestion
    override fun migrate() = flyway.migrate()
```

Reply:
Infelizmente não funciona nesse caso, pois a função `flyway.migrate()` não retorna `Unit`. Nesse caso ficaria ainda mais complexo do que utilizar as chaves diretamente.